src/components/table/table_footer.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 export const EuiTableFooter = ({ children, className, ...rest }) => {
   return (
     <tfoot className={className} {...rest}>
-      <tr className="euiTableFooterRow">{children}</tr>
+      <tr>{children}</tr>
```

Comment:
Just a note that snapshot tests will likely need to be updated as a result of this (I'm happy to do this after we merge)

Reply:
I ran the updater, but nothing updated.